WAW.API/Chat/Domain/Models/ChatRoom.cs

```diff
@@ -0,0 +1,10 @@
+using WAW.API.Shared.Domain.Model;
+
+namespace WAW.API.Chat.Domain.Models;
+
+public class ChatRoom : BaseModel {
+  public DateTime Date { get; set; }
+
+  //Relationship
+  public IList<Message> Messages { get; set; } = new List<Message>();
+}
```

Comment:
We should also add here a relationship with many `User`s, because a chat room might have one or more participants.